documentation/asciidoc/accessories/camera/filters.adoc

```diff
@@ -32,40 +32,39 @@ image::images/hoyacm500.png[CM500 Transmission Graph,width="65%"]
+Both the High Quality Camera and Global Shutter Camera contain an IR filter to reduce the camera's sensitivity to infrared light and help outdoor photos look more natural. However, you may remove the filter to:
 
-=== Filter Removal
-
-The filter removal procedure applies to both the HQ and GS cameras. The HQ camera is shown in the demonstration below.
+* Enhance colours in certain types of photography, such as images of plants, water, and the sky
+* Provide night vision in a location that is illuminated with infrared light
 
-WARNING: Before proceeding read through all of the steps and decide whether you are willing to void your warranty. *Do not proceed* unless you are sure that you are willing to void your warranty.
+=== Filter Removal
 
 WARNING: *This procedure cannot be reversed:* the adhesive that attaches the filter will not survive being lifted and replaced, and while the IR filter is about 1.1mm thick, it may crack when it is removed. *Removing it will void the warranty on the product*. Nevertheless, removing the filter will be desirable to some users.
 
+You can remove the filter from both the HQ and GS cameras. The HQ camera is shown in the demonstration below.
+
 image:images/FILTER_ON_small.jpg[width="65%"]
 
-Work in a clean and dust-free environment, as the sensor will be exposed to the air. Unscrew the two 1.5 mm hex lock keys on the underside of the main circuit board. Be careful not to let the washers roll away.
+NOTE: Make sure to work in a clean and dust-free environment, as the sensor will be exposed to the air. 
 
+1. Unscrew the two 1.5 mm hex lock keys on the underside of the main circuit board. Be careful not to let the washers roll away.
++
 image:images/SCREW_REMOVED_small.jpg[width="65%"]
-
-There is a gasket of slightly sticky material between the housing and PCB which will require some force to separate. You may try some ways to weaken the adhesive, such as a little isopropyl alcohol and/or heat (~20-30 C). 
-
-Once the adhesive is loose, lift up the board and place it down on a very clean surface. Make sure the sensor does not touch the surface.
-
+2. There is a gasket of slightly sticky material between the housing and PCB which will require some force to separate. You may try some ways to weaken the adhesive, such as a little isopropyl alcohol and/or heat (~20-30 C). 
+3. Once the adhesive is loose, lift up the board and place it down on a very clean surface. Make sure the sensor does not touch the surface.
++
 image:images/FLATLAY_small.jpg[width="65%"]
-
-Turn the lens mount around so that it is "looking" upwards and place it on a table.
-
+4. Turn the lens mount around so that it is "looking" upwards and place it on a table.
```

Comment:
Suggestion: scare quotes are against the style guide (not your fault here, of course). Maybe say "Face the lens upwards and place the mount on a flat surface" instead?